packages/ui/src/theme/Color.ts

```diff
@@ -45,12 +45,16 @@ export enum Color {
   Dark100 = '#8F949E',
   Dark200 = '#6A707C',
   Dark300 = '#5B6371',
+  /**@deprecated */
```

Comment:
@timurhamitov Can we add what should we use instead?

Reply:
@doniyor2109 These colors also will be changed in Figma and designers will use only not deprecated ones. 
And there is actually no replacement we just stop using them. Not sure if we should recommend any other color here. What do you think?